### PR TITLE
Stellar tr language support

### PR DESCRIPTION
**Pull request by erdemsahin**

### CODE REVIEW

File: tr.html [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+		<title>Stellar Global</title>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+		<link rel="stylesheet" href="assets/css/main.css" />
+		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
+		<meta property="og:title" content="Stellar Global" />
+		<meta property="og:type" content="website" />
+		<meta property="og:url" content="https://stellar-global.org/" />
+		<meta property="og:image" content="https://raw.githubusercontent.com/Stellar-Global/website/main/images/logo.png" />
+		<meta property="og:description" content="A Perfect Source about Stellar Network." />
```
Comment (MatejMecka) [inline tr.html:18]: This needs to be translated as well

File: tr.html [hunk 2]
```diff
@@ -0,0 +1,380 @@
+<!DOCTYPE HTML>
+<!--
+	Dimension by HTML5 UP
+	html5up.net | @ajlkn
+	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
+-->
+<html lang="en">
```
Comment (MatejMecka) [inline tr.html:7]:
```suggestion
<html lang="tr">
```

If this is the right iso code add this one otherwise change it

File: tr.html [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+		<link rel="stylesheet" href="assets/css/main.css" />
+		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
+		<meta property="og:title" content="Stellar Global" />
+		<meta property="og:type" content="website" />
+		<meta property="og:url" content="https://stellar-global.org/" />
+		<meta property="og:image" content="https://raw.githubusercontent.com/Stellar-Global/website/main/images/logo.png" />
+		<meta property="og:description" content="A Perfect Source about Stellar Network." />
+		<meta property="og:locale" content="en_GB" />
```
Comment (MatejMecka) [inline tr.html:19]:
```suggestion
		<meta property="og:locale" content="tr_TR" />
```

lang_Locale change it if i am wrong

File: tr.html [hunk 4]
```diff
@@ -0,0 +1,380 @@
...
+					<footer id="footer">
+						<div class="contact">
+							<ul class="icons">
+								<li><a href="https://twitter.com/stellarglobal_" class="icon brands fa-twitter" alt="Twitter" aria-label="Twitter"><span class="label">Twitter</span></a></li>
+								<li><a href="mailto:stellarglobal@protonmail.com" class="icon fa-envelope" alt="Contact" aria-label="Contact"><span class="label">Contact</span></a></li>
+								<li><a href="https://github.com/Stellar-Global/" class="icon brands fa-github" alt="GitHub" aria-label="GitHub"><span class="label">GitHub</span></a></li>
+							</ul>
+						</div>
+						<div class="languages">
+						<p><a href="https://stellar-global.org/tr.html" aria-label="Türkçe olarak değiştir.">🇹🇷</a> <a href="kr.html" aria-label="한국어로 전환">🇰🇷</a>  <a href="es.html" aria-label="Cambiar a español">🇪🇸</a> <a href="jp.html" aria-label="日本語に切り替える">🇯🇵</a></p>
```
Comment (MatejMecka) [inline tr.html:361]: You don't need to link the Turkish page back here. Also where's the English link?

**Review by MatejMecka — CHANGES_REQUESTED** [review]
Couple of changes needed

**Comment by erdemsahin** [comment]
Changes completed

**Comment by erdemsahin** [comment]
Needed Changes Completed

**Pull request merged**